purchase_work_acceptance_evaluation/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module introduces the ability to evaluate the results by work acceptance evaluation. Such as,
+evaluate of the transportation, products and services. You can also add the evaluation master data
+at configuration.
```

Comment:
This module extend Purchase Work Acceptance by adding the ability to evaluate quality of work.
User can also add as many evaluation criteria as he/she wanted in configuration menu.